NEWS.md

```diff
@@ -1,3 +1,16 @@
+# devtools 1.10.0.9001
+
+## New features
```

Comment:
Okie dokie. Is there a "contributing to devtools" page somewhere that could tell people about these little details? E.g., how to add to `NEW.md`, or change the version number?